test/suite/test_tiered16.py

```diff
@@ -80,6 +80,10 @@ def test_remove_shared(self):
             c["a"] = "a"
             c["b"] = "b"
             c.close()
+            c2 = self.session.open_cursor(uri_b)
+            c2["a"] = "a"
+            c2["b"] = "b"
+            c2.close()
```

Comment:
The commit says "Don't expect an object from a switch when the file is not yet modified." That seems reasonable. But does this mean that the test is no longer verifying that a forced flush creates a new object (i.e., on a table that isn't new)?

Reply:
```test_tiered04``` is the primary test to ensure the correct behaviour here. [These lines test](https://github.com/wiredtiger/wiredtiger/blob/b277962a80bc0dd7ba02ea23e7af44a57a7c3136/test/suite/test_tiered04.py#L222-L232) that a forced tiered flush after a regular tiered flush actually switches the files. It is not super obvious, so I added another test here in the following commit:
https://github.com/wiredtiger/wiredtiger/pull/9768/commits/3e99778e9e1935fa86c050c810edeb4c4cf5b78e

Reply:
Thanks!